### PR TITLE
Composer libraries requirements update for PHP >=8.0 support. resolves #18

### LINKED ISSUE #18: PHP 8 support?
Need php8 support, but composer.json file has php set to ^7.2
Seems to update of composer dependencies will be enough.

### PR DESCRIPTION
**Pull request by xklid101**
Dropped PHP 7 support.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -9,16 +9,16 @@
     "type": "library",
     "license": "MIT",
     "require": {
-        "php": "^7.2",
+        "php": "^8.0",
         "codeception/codeception": "^4.0",
-        "codeception/lib-innerbrowser": "^1.0",
+        "codeception/lib-innerbrowser": "^2.0",
         "slim/psr7": "^1.1",
```
Comment (szhajdu) [inline composer.json]: Please also set to ^1.3. PHP 8 support came with that version.

File: composer.json [hunk 2]
```diff
@@ -9,16 +9,16 @@
...
         "slim/slim": "^4.2"
```
Comment (szhajdu) [inline composer.json]: Please set to ^4.7 as well.

**Review by szhajdu — CHANGES_REQUESTED** [review]
Thank you for your contribution, please check my comments.

**Comment by xklid101** [comment]
Made requested chages (in 5a2e0ea91faf76cd42c169f34e37f9e507f27de0), should be ok now. You can check it out. Thanks.

**Comment by xklid101** [comment]
Just info, that new commit 8d2d586d5deda543ddf2c9ca5e104abfa3e905c0 has fixed github static analysis actions and its php8 requirement

**Pull request merged**